library/Icingadb/Widget/ItemList/BaseDowntimeListItem.php

```diff
@@ -131,24 +132,25 @@ protected function assembleTitle(BaseHtmlElement $title)
             $link = $this->createServiceLink($this->item->service, $this->item->service->host, true);
         }
 
+        if ($this->item->is_flexible) {
+            $template = t('{{#downtime}}Flexible Downtime{{/downtime}} for %s');
+        } else {
+            $template = t('{{#downtime}}Fixed Downtime{{/downtime}} for %s');
+        }
+
         if ($this->getNoSubjectLink()) {
-            $title->addHtml(new HtmlElement(
-                'span',
-                Attributes::create(['class' => 'subject']),
-                Text::create($this->item->is_flexible ? t('Flexible Downtime') : t('Fixed Downtime'))
+            $title->addHtml(TemplateString::create(
+                $template,
+                ['downtime' => HtmlElement::create('span')],
```

Comment:
The span is missing the `subject` class, please re-add that.

Reply:
Oh, thanks. Missed that.
✅